### PR TITLE
Pyzhou/tdi 45160 add encording t file unarchive

### PR DESCRIPTION
**Pull request by pyzhou-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-45160

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.components.libs/libs_src/talendzip/src/main/java/com/talend/compress/zip/Unzip.java [hunk 1]
```diff
@@ -112,6 +117,7 @@ public void doUnzipWithAes() throws Exception {
 
 		for (int i = 0; i < fileHeaderList.size(); i++) {
 			FileHeader fileHeader = (FileHeader) fileHeaderList.get(i);
+//            fileHeader.getExtraDataRecords().get(0)
```
Comment (wwang-talend) [inline main/plugins/org.talend.designer.components.libs/libs_src/talendzip/src/main/java/com/talend/compress/zip/Unzip.java:120]: not useful here?

**Comment by wwang-talend** [comment]
now default use utf8 encoding, if create a new component, it's sure right. But as before using the run platform default encoding as default, maybe ok as studio platform use utf8 default as run encoding. need to tuj run.

**Review by wwang-talend — APPROVED** [review]

**Comment by pyzhou-talend** [comment]
Useless comment removed. And I didn't change the default encoding. A check-box of "Set encoding for zip file" and uncheck by default to ensure the same behavior of the old jobs.

**Comment by pyzhou-talend** [comment]
I will recheck and maybe refactor to ensure old behavior. Thanks for the comment.

**Pull request merged**